packages/babel-parser/test/fixtures/jsx/errors/_no_plugin/input.js

```diff
@@ -0,0 +1 @@
+<div></div>
```

Comment:
Can you also add the following test case?
```jsx
<div>{name}</div>
```

Reply:
Done and done (: